### PR TITLE
Add recipe for biodigest

### PR DESCRIPTION
**Pull request by lspindler2509**
added a bioconda recipe for the pypi package biodigest 

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

### General instructions

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

### Instructions for avoiding API, ABI, and CLI breakage issues
Conda is able to record and lock (a.k.a. pin) dependency versions used at build time of other recipes.
This way, one can avoid that expectations of a downstream recipe with regards to API, ABI, or CLI are violated by later changes in the recipe.
If not already present in the meta.yaml, make sure to specify `run_exports` (see [here](https://bioconda.github.io/contributor/linting.html#missing-run-exports) for the rationale and comprehensive explanation).
Add a `run_exports` section like this:

```yaml
build:
  run_exports:
    - ...

```

with `...` being one of:

| Case                             | run_exports statement                                               |
| -------------------------------- | ------------------------------------------------------------------- |
| semantic versioning              | `{{ pin_subpackage("myrecipe", max_pin="x") }}`     |
| semantic versioning (0.x.x)      | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}`   |
| known breakage in minor versions | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| known breakage in patch versions | `{{ pin_subpackage("myrecipe", max_pin="x.x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| calendar versioning              | `{{ pin_subpackage("myrecipe", max_pin=None) }}`    |

while replacing `"myrecipe"` with either `name` if a `name|lower` variable is defined in your recipe or with the lowercase name of the package in quotes.

### Bot commands for PR management

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by lspindler2509** [comment]
@bioconda/core.

**Comment by lspindler2509** [comment]
@BiocondaBot please add label

**Comment by lspindler2509** [comment]
@BiocondaBot please fetch artifacts

File: recipes/biodigest/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+test:
+  imports:
+    - biodigest
+    - biodigest.evaluation
+    - biodigest.evaluation.d_utils
+    - biodigest.evaluation.mappers
+
+about:
+  home: "http://pypi.python.org/pypi/biodigest/"
+  license: LICENSE
```
Comment (kbseah) [inline recipes/biodigest/meta.yaml:55]: This should be `GPL-3.0-or-later`, according to the license file in the source code https://github.com/bionetslab/digest

The `license_family` field should be updated too

File: recipes/biodigest/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  license_family: OTHER
+  license_file: 
+  summary: "In silico Validation of Disease and Gene Sets, Clusterings or Subnetworks (DIGEST)"
+  doc_url: 
+  dev_url: 
+
+extra:
+  recipe-maintainers:
+    - your-github-id-here
```
Comment (kbseah) [inline recipes/biodigest/meta.yaml:64]: Could you add your github id here? otherwise this can be deleted

File: recipes/biodigest/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation"
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('biodigest', max_pin="x.x") }}
+
+requirements:
+  host:
```
Comment (kbseah) [inline recipes/biodigest/meta.yaml:20]: I think only `python` and `pip` need to be under `host`

**Review by kbseah — CHANGES_REQUESTED** [review]
Thank you very much for your contribution!

I've left a few comments in-line

File: recipes/biodigest/meta.yaml (hunk 3 above)
Comment (lspindler2509) [reply recipes/biodigest/meta.yaml:20]: thank you, I deleted the rest.

File: recipes/biodigest/meta.yaml (hunk 1 above)
Comment (lspindler2509) [reply recipes/biodigest/meta.yaml]: thanks, I updated it.

File: recipes/biodigest/meta.yaml (hunk 2 above)
Comment (lspindler2509) [reply recipes/biodigest/meta.yaml]: I did, thanks.

**Review by kbseah — APPROVED** [review]

**Pull request merged**